t/integration/test_backend.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import, unicode_literals
+
+import os
+
+from case import skip
+
+from celery.backends.azureblockblob import AzureBlockBlobBackend
+
+
```

Comment:
Although this test has certainly value, it is better if you also add and activate the integration suite by changing the [tox](https://github.com/celery/celery/blob/master/tox.ini#L43) configuration. You simply need to specify `TEST_BROKER` and `TEST_BACKEND` variables accordingly.

Reply:
Done in f17c3b6. Will verify on Travis and ping you once it's ready to be reviewed.